tests/__init__.py

```diff
@@ -0,0 +1,3 @@
+from pathlib import Path
+
+PACKAGING_EXAMPLES_DIR = Path(__file__).parent / "ethpm" / "packaging-examples"
```

Comment:
Our test modules should not have `__init__` files (and thus they won't be proper modules.  You can put this in a `tests/conftest.py` and expose it as a fixture if you need access to the variable in tests.